addons/aws/1.0.1/kubeadm-init-config-v1beta2.yml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: kubeadm.k8s.io/v1beta2
+kind: InitConfiguration
+metadata:
+  name: kubeadm-init-configuration
+nodeRegistration:
+  name: $NODE_NAME
+  kubeletExtraArgs:
+    node-ip: $PRIVATE_ADDRESS
```

Comment:
this is already set [here](https://github.com/replicatedhq/kURL/blob/e7dc0cda408ed79da1d7f7411f02ddf4ca1368a0/scripts/kustomize/kubeadm/init-orig/kubeadm-init-config-v1beta2.yml#L22). if its not necessary remove